requirements.txt

```diff
@@ -10,3 +8,5 @@ requests==2.24.0
 urllib3==1.25.10
 Werkzeug==1.0.1
 python-dotenv==0.12.0
+dateutil
```

Comment:
This should be python-dateutil

Reply:
You are also missing selenium from this file

Reply:
Added